frontend/src/pages/Compare.tsx

```diff
@@ -109,7 +109,7 @@ export default function Compare(props: PageProps) {
       // Clear the banner unless the V1 page is shown, as that page handles its own banner state.
       updateBanner({});
     }
-  }, [compareVersion, isError, error, updateBanner, runIds.length]);
+  }, [compareVersion, isError, error, isLoading, updateBanner, runIds.length]);
```

Comment:
Added this dependency as per this discussion: https://github.com/kubeflow/pipelines/pull/7844#discussion_r894884467.